# Battery message

Header header

float32 voltage
float32 charge_voltage
float32 charge_current